README.md

```diff
@@ -52,6 +52,13 @@ Production-ready features for your Symfony application. Actuator endpoints let y
         "SecurityBundle": "Symfony\\Bundle\\SecurityBundle\\SecurityBundle",
         "ActuatorBundle": "Akondas\\ActuatorBundle\\ActuatorBundle"
       }
+    },
+    "database": {
+      "default": {
+        "type": "stgreSQL100",
```

Comment:
you mean `postgreSQL100`?

Reply:
I've copied the output of my dev setup.. I assume it's an internal identifier